packages/zosfiles/__tests__/__system__/cli/delete/uss/uss.files.delete.directory.system.test.ts

```diff
@@ -0,0 +1,152 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Imperative, Session } from "@brightside/imperative";
+import { Create } from "../../../../..";
+import { inspect } from "util";
+import { runCliScript } from "../../../../../../../__tests__/__src__/TestUtils";
+import { TestEnvironment } from "../../../../../../../__tests__/__src__/environment/TestEnvironment";
+import { ITestEnvironment } from "../../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { TestProperties } from "../../../../../../../__tests__/__src__/properties/TestProperties";
+import { ITestSystemSchema } from "../../../../../../../__tests__/__src__/properties/ITestSystemSchema";
+import { appendFile } from "fs";
+
+let REAL_SESSION: Session;
+// Test Environment populated in the beforeAll();
+let TEST_ENVIRONMENT: ITestEnvironment;
+let TEST_ENVIRONMENT_NO_PROF: ITestEnvironment;
+let systemProps: TestProperties;
+let defaultSystem: ITestSystemSchema;
+let ussname: string;
+let user: string;
+
+describe("Delete Directory", () => {
+
+    // Create the unique test environment
+    beforeAll(async () => {
+        TEST_ENVIRONMENT = await TestEnvironment.setUp({
+            tempProfileTypes: ["zosmf"],
+            testName: "zos_delete_uss_directory"
+        });
+
+        systemProps = new TestProperties(TEST_ENVIRONMENT.systemTestProperties);
+        defaultSystem = systemProps.getDefaultSystem();
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(TEST_ENVIRONMENT);
+        user = defaultSystem.zosmf.user.trim();
+        ussname = `${defaultSystem.zosmf.user.trim()}.aTestUssDirectory`;
+        ussname = ussname.replace(/\./g, "");
```

Comment:
I didn't exactly follow why you replace `.` after adding one in the line above. Probably not a problem but wanted to understand the rationale

Reply:
Hmm.. Neither do I. I reckon I've copied it from the dataset delete along with the overall flow and didn't ask myself why. My bad.